docs/sdk/tutorials/plugins_example.md

```diff
@@ -467,9 +426,6 @@ Deactivate the plugin on a certain project (the plugin can still be active for o
 kili.deactivate_plugin_on_project(plugin_name=plugin_name, project_id=project_id);
 ```
 
-    Plugin Plugin bbox count deactivated on project cldlyo2bq61if0jn9efvn3soo
```

Comment:
we don't want to keep those outputs?

Reply:
I don't find them very useful, especially showing random project Ids, but I can keep them if you think they are important